lib/assets/images.dart

```diff
@@ -6,32 +6,39 @@ import 'dart:convert' show base64;
 import 'package:flame/flame.dart';
 
 class Images {
-  Map<String, ImageAssetLoader> loadedFiles = {};
+  final Map<String, _ImageAssetLoader> _loadedFiles = {};
 
   void clear(String fileName) {
-    loadedFiles.remove(fileName);
+    _loadedFiles.remove(fileName);
   }
 
   void clearCache() {
-    loadedFiles.clear();
+    _loadedFiles.clear();
+  }
+
+  Image fromCache(String fileName) {
+    final image = _loadedFiles[fileName];
+    assert(image?.loadedImage != null,
+        'Tried to access an inexistent entry on cache "$fileName"');
```

Comment:
1. trailing commas :p 
2. I think this message could be more clear by mentioning you must call `.load` before using this.